tests/literature/test_fetch_arxiv_batch.py

```
@@ -0,0 +1,63 @@
+import json
+import pytest
+from pathlib import Path
+from datetime import datetime
+datetime
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)